Public/Src/Engine/Processes/SandboxedProcessPipExecutor.cs

```diff
@@ -1605,9 +1605,24 @@ private bool PrepareFileAccessMonitoringCommon(LoggingContext loggingContext)
                 }
             }
 
-            // Untrack the globally untracked paths specified in the configuration 
+            // Untrack the globally untracked paths specified in the configuration     
             foreach (var path in m_sandboxConfig.GlobalUnsafeUntrackedScopes)
             {
+                // translate the path and untrack the translated one                
+                if (m_fileAccessManifest.DirectoryTranslator != null)
+                {
+                    var pathString = path.ToString(m_pathTable);
+                    var translatedPathString = m_fileAccessManifest.DirectoryTranslator.Translate(pathString);
+                    var translatedPath = AbsolutePath.Create(m_pathTable, translatedPathString);
+
+                    if (!path.Equals(translatedPath))

```

Comment:
>path.Equals(translatedPath)) [](start = 25, length = 28)

Do we care about case sensitiveness?

Reply:
According to Aleks' comment, the AbsolutePath comparation will take care of platform specific case sensitiveness.

Reply:
I believe that's true, but it's worth double checking

Reply:
> path.Equals(translatedPath)) [](start = 25, length = 28)
> 
> Do we care about case sensitiveness?

I checked d: equals to D: